src/components/Picker/Picker.styles.ts

```diff
@@ -25,5 +36,35 @@ export const getClassNames = memoizeFunction((classNameProps: PickerClassNamePro
     suggestionItem: {
       width: '100%',
     },
+    suggestionsBox: {
+      selectors: {
+        '.ms-Suggestions-title': {
+          color: theme.palette.themePrimary,
+          borderBottomColor: theme.palette.neutralLight,
+        },
+        '.ms-Suggestions-container': {
+          borderBottomColor: theme.palette.neutralLight,
+        },
+      },
+    },
+    suggestionsList: {
+      selectors: {
+        '.ms-Suggestions-item': {
+          selectors: {
+            ':hover': {
+              backgroundColor: theme.palette.neutralLighter,
+            },
+          },
+        },
+        '[aria-selected="true"]>.ms-Suggestions-item': {
```

Comment:
@swese44 Whoops. I just pushed this selector change not considering how it would affect the snapshot changes you just pushed.

Decided to use `aria-selected` attribute to determine whether we're looking at the currently-selected picker item rather than the classname in order to be consistent with my other selector in Dropdown, and because that seems a more safe selection assumption than the classname.

Do you prefer this? If so, I'll update the snaps myself. If not, or if you don't care one way or the other, I'll just revert.

Reply:
Meh. I'm just going to revert.